pkg/networkmanager/interfaces.js

```diff
@@ -2421,7 +2425,7 @@ PageNetworkInterface.prototype = {
             $('#network-interface-mac').text(mac);
         }
 
-        this.device_onoff.onoff("disabled", !iface);
+        this.device_onoff.onoff("disabled", !!(!iface || (dev && dev.State == 20)));
```

Comment:
and I think the boolean expression could use a comment
if you add one here, I can insert during merge